datajunction-server/datajunction_server/internal/authentication/basic.py

```diff
@@ -71,18 +71,21 @@ async def parse_basic_auth_cookie(
         or False
     )
     _logger.info("Attempting to get basic authenticated user from request cookie")
-    jwt = None
+    auth_token = None
     try:
-        jwt = get_jwt(request=request)
-    except (JWTError, AttributeError):
-        pass
-    encrypted_username = jwt.get("sub") if jwt else None
-    username = decrypt(encrypted_username) if encrypted_username else None
+        if any(request.url.path.startswith(endpoint) for endpoint in SSE_ENDPOINTS):
+            auth_token = request.query_params.get("token")
+        else:
+            auth_token = await get_token(request.headers.get("Authorization"))
```

Comment:
Maybe we should just always allow passing a token via query params?

Reply:
The `DJHTTPBearer` that I added is now more flexible and can accept either a jwt cookie or a bearer token. I found out how to make the client side event source aware of the http-only cookie so now this actually works identical across the board (no need to treat `SSE_ENDPOINTS` specially). I'll follow-up with a PR that allows for the token as a query param...any suggestions on the best way to do that welcome. :)